pom.xml

```diff
@@ -27,13 +27,7 @@
         <email>bhacker@nowhere.net</email>
       </developer>
     </developers> -->
-
-    <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
-        <connection>scm:git:https://github.com/${gitHubRepo}</connection>
-        <developerConnection>scm:git:https://github.com/${gitHubRepo}</developerConnection>
-        <tag>${scmTag}</tag>
-        <url>https://github.com/${gitHubRepo}</url>
-    </scm>    <properties>
```

Comment:
any particular reason this was removed?

not that you would need it bacause you most likely will never be be releasing this, but it does not harm things to have it here, plus it is also used to put information into the [plugins manifiest](https://github.com/jenkinsci/maven-hpi-plugin/blob/533f4d29d23b9e69c223d3897bed467b243a0300/src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractJenkinsManifestMojo.java#L216).

Reply:
Only removed as am getting error 'Attribute child.scm.connection.inherit.append.path is not allowed here'